### ISSUE TITLE
Holiday theme Cuckoo Clock Chimes

### ISSUE THREAD

**Issue by CCOSTAN** [CCOSTAN/Home-AssistantConfig#984]
Something to think about.

**Comment by thejeffreystone** [comment]
I like this idea for the hunted house automation I did. Instead of having it happen that happened at a specific time spread the fun out over the whole day. 

Also, birthdays. Might be able to do some specific things for the birthday person, like my kid who is all in on dinosaurs could have dinosaur sounds for the chime all day. 

Not to mention just changing it to something like Windsor chimes on occasion. Bells for Christmas, 

Going to have to think through how to do this so you can have multiple chime options and one automation instead of separate automations. Although that would be the easiest.

**Comment by CCOSTAN** [comment]
Yeah.. also just tracking down the actual sounds..  If you find them, I'll post/host them on the repo with the other ones. 

Birthday Idea is GREAT!

**Comment by CCOSTAN** [comment]
Quick research note: Home Assistant already supports `media_player.play_media`, so a holiday-themed cuckoo-clock chime can be driven as normal media playback. There are also public fireworks sound libraries available that should make the 4th of July variation easy to source.

Sources:
- https://www.home-assistant.io/actions/media_player.play_media/
- https://pixabay.com/sound-effects/search/fireworks/

This fits the existing holiday cuckoo-clock chimes idea, so I’m keeping the discussion on this issue instead of opening a duplicate.

<blockquote><img src="https://www.home-assistant.io/images/default-social.png" width="48" align="right"><div><img src="/images/favicon-192x192.png" height="14"> Home Assistant</div><div><strong><a href="https://www.home-assistant.io/actions/media_player.play_media/">Play specified media</a></strong></div><div>Plays specific media on a media player.</div></blockquote>
<blockquote><div><strong><a href="https://pixabay.com/sound-effects/search/fireworks/">Just a moment...</a></strong></div></blockquote>

**Comment by CCOSTAN** [comment]
Implementation pass prepared for the holiday cuckoo-clock idea:

- Added a patriotic `july-clock` sound pack with 12 hourly chime files.
- Routed the cuckoo clock to use that pack on July 3, July 4, Memorial Day, and Labor Day.
- Kept October and December seasonal packs as-is, with the default cuckoo pack as fallback.
- Updated the relevant README references so the patriotic pack is discoverable.

Validation: Home Assistant config check passes.

**Comment by CCOSTAN** [comment]
Follow-up: the patriotic cuckoo pack now has a Home Assistant-local playback path.

- Mirrored the approved `july-clock` WAVs under `config/www/sounds/july-clock` so they are served by Home Assistant at `/local/sounds/july-clock/`.
- Updated the patriotic branch of `CucKoo_Clock.yaml` to use that local path for July 3, July 4, Memorial Day, and Labor Day instead of fetching the new files from GitHub raw.
- Kept the existing October, December, and default cuckoo packs on their current paths for now to avoid duplicating the older WAV libraries.

Validation: all 12 mirrored WAVs match the approved source files, all 12 local URLs return WAV content, the Chromecast target accepted `media_player.play_media` for the local WAV URL and reported `playing`, the YAML DRY check passes, and the Home Assistant config check passes.

**Comment by CCOSTAN** [comment]
Correction/follow-up to the local playback note: all cuckoo clock packs are now moved to the Home Assistant-local playback path, not just the patriotic pack.

- `cuckoo-clock`, `july-clock`, `october-clock`, and `december-clock` now live under `config/www/sounds/` as the single source of truth for clock audio.
- `CucKoo_Clock.yaml` now uses one local base URL for every seasonal/default chime: `/local/sounds/<pack>/cuckoo-clock-XX.wav`.
- The old `config/sounds` location no longer owns the cuckoo clock packs; it remains for unrelated non-clock alert/speech audio.

Validation: all 48 cuckoo clock WAV URLs return WAV content through Home Assistant, the YAML DRY check passes, and the Home Assistant config check passes.

**Comment by CCOSTAN** [comment]
Follow-up after first live use: the patriotic/fireworks cuckoo branch is temporarily disabled.

Reason: the fireworks-style chimes are too startling for a noise-sensitive pet at home. The `july-clock` assets remain in place under `config/www/sounds/`, but the date-selection branch in `CucKoo_Clock.yaml` is commented out so July 3/4, Memorial Day, and Labor Day fall back to the normal cuckoo pack for now.

Revisit later: re-enable the commented patriotic date branch when we are ready to test a quieter version or use the pack again.
